sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ExpressionSet.scala

```diff
@@ -59,6 +59,12 @@ class ExpressionSet protected(
     }
   }
 
+  def addMultiExpressions(elems: Set[Expression]): ExpressionSet = {
```

Comment:
Override `++` instead of `addMultiExpressions`? As it is a `Set`, we better follow Set's semantics.